org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/workspace/WorkspaceProvider.xtend

```diff
@@ -52,25 +53,41 @@ class WorkspaceProvider {
 	def String read(String resourcePath) {
 		return Files.asCharSource(getWorkspaceFile(resourcePath), UTF_8).read
 	}
+	
+	def boolean isLocalBackupFile(String resourcePath) {
+		if (resourcePath === null) {
+			return false
+		} else {
+			val pattern = Pattern.compile('.*\\.' + BACKUP_FILE_SUFFIX + '(_\\d+)?\\.[^.]+')
```

Comment:
I guess, since the regex pattern is static, it could be factored out instead of recreating it on every method invocation…